libraries/AP_RangeFinder/AP_RangeFinder_Benewake_TFMiniPlus.cpp

```diff
@@ -124,7 +124,7 @@ bool AP_RangeFinder_Benewake_TFMiniPlus::init()
 
     hal.scheduler->delay(100);
 
-    _dev->register_periodic_callback(10000,
+    _dev->register_periodic_callback(30000,
```

Comment:
would be good to have a comment here about the issue. It's not common to tell the hardware to oversample and read at 1/3 the sample rate. This also means we will end up having some imprecision when plotting the distance over time as the time passed is not always the delta between measurements.  However I think it's a good enough solution. Just add a comment about the issue, which could also be solved in tfmini's firmware.

Reply:
Thank you for sharing your concerns.
Just in case, I sent the feedback to Benewake.